Source/WebKit/WPEPlatform/wpe/drm/WPEDisplayDRM.cpp

```diff
@@ -100,37 +104,29 @@ static std::optional<std::pair<UnixFileDescriptor, CString>> findDevice(struct u
         if (g_strcmp0(deviceSeatID, seatID))
             continue;
 
-        bool isbootVGA = false;
+        bool isBootVGA = false;
         if (auto* pciDevice = udev_device_get_parent_with_subsystem_devtype(udevDevice.get(), "pci", nullptr)) {
             const char* id = udev_device_get_sysattr_value(pciDevice, "boot_vga");
-            isbootVGA = id && !strcmp(id, "1");
+            isBootVGA = id && !strcmp(id, "1");
         }
 
-        if (!isbootVGA && drmDevice)
+        if (!isBootVGA && drmDevice)
```

Comment:
I dont see drmDevice being assigned in the new version of the code.

Reply:
... so this if() ia never satisfied

Reply:
Right. I forgot to replace it, thanks!